### PR TITLE
add/change registry and remote branch parameters

### PR DESCRIPTION
**Pull request by lolodi**
added remote-branch
added registry-resource_id

removed registry-resource-group

parameters

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@lolodi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (lolodi). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by lolodi** [comment]
@derekbekoe

File: src/command_modules/azure-cli-vsts/azure/cli/command_modules/vsts/custom.py [hunk 1]
```diff
@@ -118,14 +125,16 @@ def _call_rp_configure_cicd(
     :type resource_group_name: String
```
Comment (derekbekoe) [inline src/command_modules/azure-cli-vsts/azure/cli/command_modules/vsts/custom.py]: The docstring for this method still has `resource_group_name` although the argument has been changed to `registry_resource_id`.

Comment (lolodi) [inline src/command_modules/azure-cli-vsts/azure/cli/command_modules/vsts/custom.py]: those are two different parameters, I'm still passing resource_group_name, what I changed is registry_resource_group-->registry_resource_id

**Pull request merged**